stl/inc/random

```diff
@@ -2287,12 +2331,17 @@ private:
         }
 
         for (;;) { // generate and reject
+            using _Uty = make_unsigned_t<_Ty>;
+            constexpr auto _Ty_max{(numeric_limits<_Ty>::max)()};
+            const auto _Ty1_max{_Float_upper_bound<_Ty1, _Uty>(static_cast<_Uty>(_Ty_max))};

```

Comment:
We should use template argument deduction for the `_Uty` argument:
```suggestion
            const auto _Ty1_max{_Float_upper_bound<_Ty1>(static_cast<_Uty>(_Ty_max))};
```
Earlier, I noted this on one line, but failed to explain that my extremely terse, easily missed "Occurs below." is our shorthand for "There are several occurrences of this issue that should be fixed; please search for them as they are highly systematic - that will be faster than commenting on each of them separately." Sorry about that!